tests/test_strain_design_deterministic.py

```diff
@@ -146,3 +160,31 @@ def test_invalid_input(self, cplex_optknock):
             optknock.run(target="EX_ac_lp_e_rp_")
         with pytest.raises(ValueError):
             optknock.run(biomass="Biomass_Ecoli_core_N_lp_w_fsh_GAM_rp__Nmet2")
+
+
+@pytest.mark.skipif('gurobi' not in solvers, reason="No gurobi interface available")
+class TestGrowthCouplingPotential:
+    def test_growth_coupling_potential_runs(self, gurobi_growth_coupling_potential):
+        _, growth_coupling_potential = gurobi_growth_coupling_potential
+        result = growth_coupling_potential.run()
+        assert "obj_val" in result
+        assert "knockouts" in result
+        assert "knockins" in result
+        assert "medium" in result
+
+    def test_growth_coupling_potential_benchmark(self, gurobi_growth_coupling_potential, benchmark):
+        _, growth_coupling_potential = gurobi_growth_coupling_potential
+        benchmark(growth_coupling_potential.run)
+
+    def test_result_is_correct(self, gurobi_growth_coupling_potential):
```

Comment:
As far as I can tell, this test does not verify the function of the method. TPI is already essential for growth and the knock-out results do not modify that situation. (Please see the attached notebook.)
[opt_couple.ipynb.gz](https://github.com/biosustain/cameo/files/3378146/opt_couple.ipynb.gz)